SpliceAI.pm

```diff
@@ -268,11 +271,29 @@ sub run {
         $hash{'SpliceAI_cutoff'} = $result_flag;
       }
 
-      return ($self->{config}->{output_format} eq "json" || $self->{config}->{rest}) ?  {SpliceAI => \%hash} : \%hash;
+      $hash_aux{$data_value->{gene}} = \%hash;
+    }
+  }
+
+  return {} unless(%hash_aux);
+
+  my $result = {};
+
+  my $n_genes = scalar keys %hash_aux;
+  if($n_genes == 1) {
```

Comment:
Should we not only report scores if the gene names match? In that case  you don't need to differentiate between n_genes == 1 or n_genes > 1. I would almost suggest a warning flag if SpliceAI has data for the variant but where the gene names don't match.